README.md

```diff
@@ -1,96 +1,44 @@
-
-## Create entity
-
-`createEntity(entity, options)`
-
-Pass a full resource object to `createEntity`. It will trigger a request to `POST /${entity.type}`.
-
-Expects the API to return `200 OK` with the newly created resource object in response body.
-
-## Read endpoint
-
-`readEndpoint(endpoint, options)`
-
-Provide the endpoint from where to read. E.g. `users/1/roles`. The module will read each entity and map them from their type specified.
-
-## Update entity
-
-`updateEntity(entity, options)`
-
-Pass your resource object to `updateEntity`. It will trigger a request to `PATCH /${entity.type}/${entity.id}`.
-
-Expects the API to return `200 OK` with the updated resource object in response body.
-
-## Delete entity
-
-`deleteEntity(entity.options)`
-
-Accept a resource object and triggers a request to `DELETE /${entity.type}/${entity.id}`.
-
-Expects the API to return `204 No content`.
-
-* * *
-
-Made with ❤️ by Team [Dixie][dixie]
-
- [dixie]: http://dixie.io
+# Introduction
+Welcome to the __Redux JSON API__ we ❤️ JSON and we love simple api's. With this in mind was this libiary born. We strive to keep this libiary as small but as powerfull as possible so it would be easy adaptable into any `JSON API` stack enviroment.
+
+We would love all of you users, to open disscusions/issues with us, that can make this tool pleasent for all to use, and ther for give a better libiary in the end. And if you build somthing with it, please let us know so we can see the cool stuff you are doing!
+
+If there is anything that you feel is not documented, or could be refered better to such as things from `redux` let us know aswell.
+
+## Table of content
+1. [Introduction](#introduction)
+2. [Getting Started](#getting-started)
+3. [How To Use](#how-to-use)
+4. [Api Overview](#api-overview)
+5. [Good reads](#good-reads)
+6. [Contribute](#contribute)
+7. [Contributors](#contributors)
+
+## [Getting Started](docs/GettingStarted.md)
+## [How to use](docs/HowToUse.md)
+## API overview
+1. [Read __createEntity()__](docs/apis/createEntity.md) - Creating new entities
+2. [Read __readEndpoint()__](docs/apis/readEndpoint.md) - Retrive data from your database
+3. [Read __updateEntity()__](docs/apis/updateEntity.md) - Update a given entity's values
+4. [Read __deleteEntity()__](docs/apis/deleteEntity.md) - Remove entity from your database
+
+## Good reads
+1. [__Redux__](http://www.github.com) - Read about redux and core principles.
+2. [__JSON API__](http://www.jsonapi.org/) - Read about the specifications for JSON API.
+
+## Contribute
+The reason for this repository is to keep on a good and healty & simple api for _JSON API_, for Redux applications weahter you use it in _react_ or any other framework of choise. Feel like getting envolved into to this, then get cloning and follow our simple guide lines, if you just want to give feedback or report bug click that [issues](https://github.com/dixieio/redux-json-api/issues) button and lets talk about it.
```

Comment:
Redundant to talk about the purpose of the library in _Contribute_ section. It has already been introduced in _Introduction_.